.github/workflows/ci.yml

```diff
@@ -1,5 +1,10 @@
 name: Build
-on: [pull_request, workflow_dispatch]
+on:
```

Comment:
Syntax docs for this scenario: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-activity-types-and-filters-with-multiple-events